packages/server/prod-server/src/netlify.ts

```diff
@@ -0,0 +1,19 @@
+import { loadServerEnv } from '@modern-js/server-core/base/node';
+import { createServerBase } from '@modern-js/server-core/base';
+import { initProdMiddlewares } from './init';
+import { BaseEnv, ProdServerOptions } from './types';
+
+export { initProdMiddlewares, type InitProdMiddlewares } from './init';
+
+export type { ProdServerOptions, BaseEnv } from './types';
+
+export const createNetlifyFunction = async (options: ProdServerOptions) => {
```

Comment:
Why add it here instead of in the netlify entry ?

Reply:
There is no substitute for Netlify entry, which uses this